src/web/mjs/connectors/MangaSail.mjs

```diff
@@ -29,69 +29,61 @@ export default class MangaSail extends Connector {
         Engine.Settings.addEventListener('saved', this._onSettingsChanged.bind(this));
     }
 
-    /**
-     *
-     */
-    _getMangaListFromPages( mangaPageLinks, index ) {
-        if( index === undefined ) {
-            index = 0;
+    async _getMangas() {
+        let mangaList = [];
+        const uri = new URL('/directory', this.url);
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, 'ul.pagination li.pager-last a');
+        const pageCount = parseInt(data[0].href.match(/([\d]+)$/)[1]);
+        for(let page = 0; page <= pageCount; page++) {
```

Comment:
Page starts at `0` and not at `1` is this correct?

Reply:
Correct. Link for page entitled "2" actually is https://www.mangasail.net/directory?page=1 and so on.